main/lsp/LSPPreprocessor.cc

```diff
@@ -229,6 +229,11 @@ unique_ptr<LSPTask> LSPPreprocessor::getTaskForMessage(LSPMessage &msg) {
                 return make_unique<SorbetResumeTask>(*config);
             case LSPMethod::SorbetError:
                 return make_unique<SorbetErrorTask>(*config, move(get<unique_ptr<SorbetErrorParams>>(rawParams)));
+            case LSPMethod::WorkspaceDidChangeConfiguration: {
+                auto tmp = this->openFilePaths();
+                return make_unique<DidChangeConfigurationTask>(
+                    *config, move(get<unique_ptr<DidChangeConfigurationParams>>(rawParams)), move(tmp), nextVersion++);
```

Comment:
Note that since `openFilePaths` is returning a `const` thing and therefore `tmp` is `const`, the `move` here is not actually moving anything and will wind up doing a copy (thanks weird C++ semantics).  You should probably change the return type of `openFilePaths`, and that will enable you to drop the temporary variable here.